tests/reference_data/methane.json

```diff
@@ -0,0 +1,11 @@
+{"1": {
+ "cell": [[0.0, 0.0, 0.0], [0.0, 0.0, 0.0], [0.0, 0.0, 0.0]],
```

Comment:
IIRC it does, yes, because this is a fairly unambiguous way of indicating a non-periodic system. Some utilities in `quippy` do choke on a unit cell of zeroes, though I can't remember whether it's a problem in ASE.

Reply:
From the [ase documentation on atoms](https://wiki.fysik.dtu.dk/ase/ase/atoms.html#module-ase.atoms) it's not entirely clear that a zero unit cell signals no periodicity - but it does state that this is the default and means the lattice is not defined.